startup/routes.js

```diff
@@ -1,8 +1,26 @@
 const express = require('express')
+const rateLimit = require('express-rate-limit')
+
+const ApiError = require('../helpers/apiError')
+const globalErrorHandler = require('../middleware/globalErrorHandler')
+
 const example = require('../routes/example')
 
 module.exports = function (app) {
   app.use(express.json())
 
-  app.use('/api/users', example)
+  const limit = rateLimit({
+    max: 100,
+    windowMs: 60 * 60 * 1000,
+    message: 'Too Many Requests',
+  })
+
+  app.use('/api', limit)
+  app.use('/api/v1/users', example)
```

Comment:
Are we going to migrate future versions for our few endpoints? We can just set it to api/users. Whatever you prefer

Reply:
Rest standard I think says that it should be /api and some version

Reply:
Okay, I talked to my friend and he told me that version are being used basically for big applications where you know that there will be more then one code version

Reply:
aha, ok